app/services/process_submission_service.rb

```diff
@@ -9,6 +11,17 @@ def perform
     submission.update_status(:processing)
     submission.responses = []
 
+    submission.submission_details.each do |submission_detail|
+      submission_detail = submission_detail.symbolize_keys
+
+      if submission_detail.fetch(:type) == 'json'
+        JsonWebhookService.new(
+          runner_callback_adaptor: Adapters::RunnerCallback.new(url: submission_detail.fetch(:url)),
+          webhook_destination_adaptor: Adapters::WebhookDestination.new(url: 'https://example.com/json_destination_placeholder')
```

Comment:
Maybe we should assign this value to a constant named JSON_DESTINATION_PLACEHOLDER so it's obvious it's wip